eve/main.yml

```
@@ -72,10 +72,14 @@ models:
         echo "Could not reach Docker daemon from buildbot worker" >&2
         exit 1'
       haltOnFailure: true
-  - SetProperties: &set_operator_properties
-      properties:
-        OPERATOR_IMAGE_NAME: registry.scality.com/sf-eng/zenko-operator
-        OPERATOR_IMAGE_TAG: '1.1.6'
+  - SetPropertyFromCommand: &set_operator_image_name
+      name: Set operator image name
+      property: OPERATOR_IMAGE_NAME
+      command: yq '."zenko-operator".sourceRegistry' solution/deps.yaml
```

missing the image name: 
```suggestion
      command: yq '."zenko-operator" | .sourceRegistry + "/" + .image' solution/deps.yaml
```

---

thanks for the catch, just added it.